test/e2e/framework/util.go

```diff
@@ -154,6 +154,21 @@ const (
 	// How long claims have to become dynamically provisioned
 	ClaimProvisionTimeout = 5 * time.Minute
 
+	// How long claims have to become bound
+	ClaimBindingTimeout = 5 * time.Minute
```

Comment:
I think most const here were 3 minutes

Reply:
Not consistent. some are 5 minutes while others are 3 minutes.
I change all to 3 minutes